src/main/resources/public/css/stylesheet.css

```diff
@@ -9,9 +9,14 @@ input, textarea { display: table-cell; }
     justify-content: center;
 }
 
-#logo {
-    font-size: 90px;
-    margin: 0;
+#icon {
+    display: block;
+    margin-left: 2.5%;
```

Comment:
The logo will be floating towards the middle as you increase the size of the screen. Please use px or em instead. (I suggest 1em)

Reply:
fixed, changed to 1em margins